proto/pocket/supplier/proof.proto

```diff
@@ -1,14 +1,14 @@
 syntax = "proto3";
 package pocket.supplier;
 
+import "cosmos_proto/cosmos.proto";
+import "pocket/session/session.proto";
+
 option go_package = "github.com/pokt-network/poktroll/x/supplier/types";
 
-// TODO_UPNEXT(@Olshansk): The structure below is the default (untouched) scaffolded type. Update
-// and productionize it for our use case.
 message Proof {
-  string index = 1;
-  string supplier_address = 2;
-  string session_id = 3;
-  string merkle_proof = 4;
+  string supplier_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
OPTIONAL ULTRA NIT: You update some (not all) of the comments for the `Claim` proto to be on the line above as opposed to after the `;`. This was not reflected i the `Proof` proto. 

We can always change it later but wanted to call it out.